docs/installation/installing-octopus/automating-installation.md

```diff
@@ -0,0 +1,108 @@
+
+Automating the installation of Octopus Server is a three step process.
+
+### 1. Install MSI on a temporary machine interactively
+In this step we install the MSI on a machine interactively so that we can complete the wizard to add a new instance.
+Follow all the steps in the wizard and once you get to the last screen, click on `Show Script` (**do not click Install**).
+![](show-script.png "width=500")
+
+Copy the script generated onto the clipboard.
+
+Here is an example what the script looks like, **you need to modify this script manually with your settings**:
+```bash
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" create-instance --instance "new instance" --config "C:\Octopus\new instance\OctopusServer-new instance.config"
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" configure --instance "new instance" --home "C:\Octopus\new instance" --storageConnectionString "Data Source=(local)\SQLEXPRESS;Initial Catalog=foo;Integrated Security=True" --upgradeCheck "True" --upgradeCheckWithStatistics "True" --webAuthenticationMode "UsernamePassword" --webForceSSL "False" --webListenPrefixes "http://localhost:80/" --commsListenPort "10943" --serverNodeName "JOHNSBOX"
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" database --instance "new instance" --create --grant "NT AUTHORITY\SYSTEM"
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" service --instance "new instance" --stop
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" admin --instance "new instance" --username "admin" --email "" --password ""
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" license --instance "new instance" --licenseBase64 "a very long license string"
+"C:\Program Files\Octopus Deploy\MyOctopus\Octopus.Server.exe" service --instance "new instance" --install --reconfigure --start --dependOn "MSSQL$SQLEXPRESS"
+```
+
+
+### 2. Install MSI in central server silently
+
+To install the MSI silently:
+
+```bash
+msiexec /i Octopus.<version>.msi /quiet RUNMANAGERONEXIT=no
+```
+
+By default, the Octopus files are installed under **%programfiles%**. To change the installation directory, you can specify:
+
+```bash
+msiexec /i Octopus.<version>.msi /quiet RUNMANAGERONEXIT=no INSTALLLOCATION="C:\YourDirectory"
+```
+
+### 3. Configuration
+
+The MSI installer simply extracts files and adds some shortcuts and event log sources. The actual configuration of Octopus Server is done later, via the script you copied above.
+
+To run the script start an admin shell prompt and paste the script onto it, this should apply all the settings to the new instance.
+
+## Desired State Configuration
+
+Octopus can also be installed via [Desired State Configuration](https://msdn.microsoft.com/en-us/powershell/dsc/overview) (DSC). Using the module from the [OctopusDSC GitHub repository](https://github.com/OctopusDeploy/OctopusDSC).
```

Comment:
Probably better to reference the package from http://powershellgallery.com/packages/OctopusDSC